apps/medon-fe/src/components/SelectTimeSlot/index.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import 'translation/i18next';
+import { ReactNode } from 'react';
+import { Provider } from 'react-redux';
+import { store } from 'redux/store';
+import { ThemeProvider } from 'styled-components';
+import { theme } from 'styles/theme';
+import SelectTimeSlot from './index';
+
+const Wrapper = ({ children }: { children: ReactNode }) => (
```

Comment:
not important, but i think you should use the Wrapper that i created, just to not have two of the same